edgelet/iotedged/Cargo.toml

```diff
@@ -18,7 +18,8 @@ sha2 = "0.7.0"
 tokio = "0.1.8"
 tokio-signal = "0.2"
 url = "1.7"
-
+url_serde = "0.2"
+native-tls = "0.2"
 hsm = { path = "../hsm-rs"}
```

Comment:
Keep the blank line that separates crates.io deps from workspace deps.

Reply:
The blank line separates crates.io deps from workspace deps. ie it goes before `hsm`.

Also, keep the deps in alphabetical order, so `native-tls` goes before `serde_json`.